qemu/tests/numa_memdev_options.py

```diff
@@ -111,10 +114,31 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment
     """
-    error_context.context("Check host's numa node(s) amount!", logging.info)
-    check_host_numa_node_amount(test)
+    error_context.context("Check host's numa node(s)!", logging.info)
+    valid_nodes = get_host_numa_node()
+    if len(valid_nodes) < 2:
+        test.cancel("The host numa nodes that whose size is not zero should be "
+                    "at least 2! But there is %d." % len(valid_nodes))
+    node1 = valid_nodes[0]
+    node2 = valid_nodes[1]
+
+    if params.get('policy_mem') != 'default':
+        error_context.context("Assign host's numa node(s)!", logging.info)
+        params['host-nodes_mem0'] = node1
+        params['host-nodes_mem1'] = node2
+
+    if params.get('set_node_hugepage') == 'yes':
+        if 'numa_hugepage' in params['shortname']:
+            params['target_nodes'] = "%s %s" % (node1, node2)
+            params['target_num_node%s' % node2] = '1546'
```

Comment:
Let's put the number in cfg, or just calculate the number by size_memX / Hugepagesize.
size_mem0 = 1024M
size_mem1 = 3072M